app/views/conferences/_form.html.slim

```diff
@@ -16,6 +16,7 @@
   = f.input :flights
   = f.input :round
   = f.input :accomodation
+  = f.input :lightningtalkslots, input_html: {value: false}, label: "Lightning Talk Slots"
```

Comment:
You don't have to assign `false` here. The default value is `nil` (which is »falsey«). In fact, forcing the value to `false` will overwrite the previous value when a conference is edited at a later time.